### DISCUSSION TITLE
MECHanic skill: Exosuit comparatively lackluster; particularly when worn by Mechanic

### DISCUSSION THREAD

**Discussion by Joel-B-Williams** [FakeFishGames/Barotrauma#13449] [Ideas and suggestions]
The Exosuit is an awesome piece of gear, however is largely overshadowed by other specialty suits or items.

Pros:
High armor/resistance (legit benefit)
Storage (light/moderate benefit - competes with Cargo Scooter and Backpack for usefulness)
Higher Depth Rating (marginal to no benefit, gameplay does not require 10k depth rating)
Assists with HMG recoil (legit benefit, but seems odd that the benefit is tied to an item from another class)

Cons:
Extremely slow (helped in case of Mechanic, specifically, but still slow)
Requires Fuel Rod
No speed benefit from Oxygenite

While the speed boost when used by a Mechanic is welcomed, it doesn't counteract the large list of downsides.
As the suit is in-lore a construction mech, adding some abilities/utility to assist traditional Mechanic duties could help to balance the overall value.

*Suggested ideas* -- some/all of these could be implemented to add utility/benefit to the suit:

_Multitool_
While wearing the suit, the user is treated as having a Wrench, Screwdriver, and Crowbar equipped.  ie; basic repairs can be completed without need for carrying tools.  Doors can be opened

_Increased light output_
The reduced view is fitting, however it would also make sense to have high-powered lights on a suit of this style.  Between a standard suit and a spotlight

_Power Arm_
Native melee attack, similar to a Heavy Wrench/buffed crowbar.  Having the _wrench_ keyword to trigger mechanic talent-buffs would make sense

_Mechanic specific buffs_
Similar to the speed increase, a Mechanic should get the most out of this suit.  Potential for a melee damage buff, plasma cutter speed/effectiveness increase, repair speed increase

**Comment by bookslayer10** [comment]
The Exosuit is supposed to be getting buffed to 80% damage resistance in a coming update. That is combined with other dive suits getting nerfs to their damage resistances, so the Exosuit is significantly tankier than many other competing options. Backpacks slow your movement speed down by 30% and scooter speed by 30%, which is much less than the Exosuit's 60% and 75%, but means that (with backpack) Makeshift armor or Abyss suits are barely faster than the Exosuit, with worse defense and the inability to carry heavy items on your back.

The Exosuit is unlocked in the Brawler skill tree of the Mechanic. This tree has talents that are mostly dedicated to just combat, not really Mechanic tasks. Also, consider the Engineer. Their Weapons Engineer skill tree has several items that really aren't useful for normal Engineer tasks. My point is, I don't think the Exosuit needs to be very good at traditional Mechanic duties in order to be a fun and effective item.

Overall, I do agree that the Exosuit should get some buffs. It should have higher husk infection resistance chance, maybe 80% or so, just to be a middle ground between the 50% of normal suits and the 90% of PUCS. It also should get the stun resistance that Makeshift Armor has. 

If it really needs to be buffed for normal Mechanic tasks, let it dual-wield two-handed tools. Welders, Cutters, Scooters, Extinguishers, Crates, Ammo Boxes, Railgun Shells, Riot Shields, etc. However, no dedicated weapons allowed. As fun as dual boomsticks or HMGs would be, that would be WAY too OP. Simply being able to weld at double speed, or carry two ammo boxes at once, would automatically make the Exosuit fantastic for Mechanic. Plus, dual wielding dive scooters would give it a slightly faster movement option. This idea would probably be OP no matter what though, so I don't really think it should be added.

If dual-wielding is okay, I'm conflicted on letting Heavy Wrenches be allowed. On one hand, they are technically a tool, and they are unlocked in the same tree as Exosuits, so it would make sense for them to synergize. Heavy Wrenches also take a lot of Mechanical skill to even swing. On the other hand, dual wielding Heavy Wrenches would be ridiculously powerful and would make the Exosuit the strongest suit for a Brawler Mechanic hands down. Does Mechanic really deserve that kind of power compared to other classes?

**Reply by Joel-B-Williams** [reply]
Agreed that the tree is the Combat Focus - my "ideal" of the above would be the built-in attack and/or melee damage buff _specifically for mechanics_
The "multitool" I think would be interesting and add utility to the item, while keeping with the lore.
Ultimately my goal here is to present a handful of options that would be, hopefully, relatively straightforward updates to XML (requires light dev work) while not getting into the OP range that carrying two HMG's would be.

To your point of speed, the Backpack is a good example of being very slightly faster, but far less armor (especially post-update, if things go as planned).  The Cargo Scooter is probably the biggest counter-point, as a non-tree talent that provides not only storage, but speed, is cheap to create & readily available.  Additionally, backpacks and cargo scooters are both low-tier basic talents; being a Tier 4 tree-talent, the Mech Suit should feel like an accomplishment over these things, not a trade-off.  The design philosophy discussed by the Devs is that the tree talents are less amazing, but the top-tier talents are meant to be the pinnacle.  

The Exo feels like a side-grade to the combo of a backpack and basic scooter.  It questionably-at-best competes with PUCS, and even the Slipsuit, which is a low-tier tree item.  This is even counting the Mechanic-specific speed increase.  None of this is in line with the dev's proposed talent-progression, where the top-tier talents are intended to be the reason for the tree.

**Reply by bookslayer10** [reply]
Firstly, I don't think Multitool is a very good idea. For one, it overloads the amount of options you have while not letting you disable any of them for precision. What if I want to take a button off of a wall but I keep opening the wiring menu? What if I want to rewire a door but I keep crowbarring it instead? It will just be too clunky and unintuitive. Plus, you won't be able to use quality bonuses, so around-the-ship repairs won't be done using the multitool. Really, this is just going to be useful on Beacon Stations, and what does it save you? Two, maybe three item slots?

Cargo scooter is good, but you do sacrifice mobility and need to fiddle with inventories if you get surprised in combat. Anyway, the storage space on the Exosuit is supposed to be a bonus to make up for the lack of move speed - if you want, you can also take along a backpack and a cargo scooter for maximum storage, nothing is really stopping you. Yes, there are earlier ways to unlock methods of bulk item transport. They key difference between those cargo storage methods and the Exosuit is that the Exosuit comes with an Exosuit to wear, not just cargo space.

PUCS is definitely a good item, which it should be, as it's unlocked on the same level as the Exosuit. When it comes to combat and exploration, PUCS isn't especially good. It has average speed, the husk resistance is good but not always useful, and I'm pretty sure it will soon have a nerf to its defense as part of the unstable changes. Slipsuit is good in caves but also is going to have much lower resistances. It's fine if you value speed over survivability and total cargo space, but most monsters will still be able to catch you no matter how fast you run, so I don't think speed is really as overall good as defense unless the mission is easy.
